### PR TITLE
Example graph queries

### PR DESCRIPTION
**Pull request by Ragsboss**
Based on customer request, creating a list of sample graph queries to help probe authors get started on the IBA probes

### CODE REVIEW

File: graph_queries/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+probes. You can certainly use these samples as starting points but it's unlikely for
+any of these to be an accurate match for your specific requirements.
+
+## Interactive query execution
+
+Apstra Operating System GUI has 'GraphQL API Explorer' accessible from the blueprint
+details view. You can execute the queries in this directory in the explorer
+and examine the results. You can also tweak the query and see the results
+interactively. Use the explorer as an aid to arrive at the final queries that work
+for your specific requirements.
```
Comment (huttmf) [inline graph_queries/README.md:35]: Might be good to specify that these example are QE queries.

File: README.md [hunk 2]
```diff
@@ -165,3 +166,10 @@ The files in this folder are IBA probe json payloads that are represented as
 [JINJA templates](http://jinja.pocoo.org/docs/2.10/templates/). You need to use
 aos-cli to load these probes on to AOS server. The command to use in aos-cli is
 `probe create --blueprint <id> --file </path/to/template/file> [<additional_args>]`
+
+
+## Probe graph queries
+
+Authoring new probes requires familiarity with graph queries to select the elements
+of interest whose telemetry is ingested into the probes. Refer to
+[graph queries](graph_queries/README.md) for more details.
```
Comment (huttmf) [inline README.md]: I think we may also need some explanation of the collectors,  or at least a reference to telemetry: https://files.apstra.com/docs/telemetry.html

File: graph_queries/README.md (hunk 1 above)
Comment (Ragsboss) [reply graph_queries/README.md:35]: That thought did cross my mind but I felt given this is an IBA repo, the queries here work for IBA and the readers don't need to be worried about other query languages we may support. Wdyt? I'll mention it in some form I guess.

File: README.md (hunk 2 above)
Comment (Ragsboss) [reply README.md]: Will do

File: graph_queries/README.md (hunk 1 above)
Comment (huttmf) [reply graph_queries/README.md:35]: I meant that if they paste the query that's shown into GraphQL API Explorer they won't work unless they select qe.

File: graph_queries/leaf interfaces connected to spines.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  match(
+    node('system', name='leaf', role='leaf', system_id=not_none()).out('hosted_interfaces')
+    .node('interface', name='leaf_intf').out('link')
+    .node('link').in_('link')
+    .node('interface', name='spine_intf').in_('hosted_interfaces')
+    .node('system', name='spine', role='spine')
+  )
+variants:
+  - >
+      Add deploy_mode check to leaf or spine to get only leafs that are actively in
```
Comment (hk-spark) [inline graph_queries/leaf interfaces connected to spines.yaml:15]: One variation here is also to get only interfaces of particular speed.

**Comment by hk-spark** [comment]
graph_queries/spine interfaces.yaml  is an empty file added ?

**Pull request merged**